imports/client/ui/pages/Home/HowToHelpSection/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Container, Row, Col, Button } from 'reactstrap'
+import Item from './Item'
+import i18n from '/imports/both/i18n/en'
+import './styles.scss'
+
+const HowToHelpI18N = i18n.Home.how_to_help_section
+
+const {
+  title,
+  content
+} = HowToHelpI18N
+
+const {
+  items
+} = content
+
+const HowToHelpSection = () => (
+  <section id='how_to_help_section'>
+    <Container>
+      <div className='title'>{title}</div>
+      <Row className="items">
+        {items.map((item, i) => (
+          <Col key={i}>
+            <Item item={item} />
+          </Col>
+        ))}
+      </Row>
+    </Container>
+    <Button className='sign-and-post' tag={Link} to='/sign-in'>Sign In and Post</Button>
```

Comment:
What if the user is already logged in?

Reply:
Verified that it should be on both sites. Confirmed with Andy that this should display the add resource/gather form. Will update.